### ISSUE TITLE
Kolmogorov-Smirnov test

### ISSUE THREAD

**Issue by jkylearmstrong** [rstudio/sparklyr#1481]
Will function be added for sparks Kolmogorov-Smirnov test?

**Comment by kevinykuo** [comment]
Yes, when Spark 2.4 is released, we'll have it implemented. https://issues.apache.org/jira/browse/SPARK-21898

**Comment by jkylearmstrong** [comment]
Will 2 sample test be supported?

**Comment by kevinykuo** [comment]
https://issues.apache.org/jira/browse/SPARK-21898
